AppServer/google/appengine/tools/devappserver2/module.py

```diff
@@ -521,6 +525,7 @@ def _error_response(self, environ, start_response, status):
     start_response('%d %s' % (status, httplib.responses[status]), [])
     return []
 
+  # AppScale.
```

Comment:
`AppScale: check if the instance should be shutting down before handling request`